### ISSUE TITLE
Need a grid view of the services

### ISSUE THREAD

**Issue by benroeder** [marcopaz/is-service-up#19]
There are too many services to fit on one page, needs a grid view

**Comment by cosenal** [comment]
Yes, that would be nice, but not really necessary once we have #5 implemented

**Comment by benroeder** [comment]
depends how many you would like to see on one screen

**Comment by benroeder** [comment]
Could sort by service status, most broken at the top
